api/envoy/config/metrics/v3/stats.proto

```diff
@@ -406,3 +406,30 @@ message HystrixSink {
   // <https://github.com/Netflix/Hystrix/wiki/Metrics-and-Monitoring#hystrixrollingnumber>`_.
   int64 num_buckets = 1;
 }
+
+// Stats configuration proto schema for ``envoy.stat_sinks.graphite_statsd`` sink.
+// The sink emits stats with `Graphite <https://graphite.readthedocs.io/en/latest/tags.html>`_
+// compatible tags. Tags are configurable via :ref:`StatsConfig
+// <envoy_v3_api_msg_config.metrics.v3.StatsConfig>`.
+// [#extension: envoy.stat_sinks.graphite_statsd]
+message GraphiteStatsdSink {
```

Comment:
For historical reasons, all of the stat sinks are here. However for this new extension, can you add it where it probably should be, which is in this tree: https://github.com/envoyproxy/envoy/tree/main/api/envoy/extensions

I would just make a dedicated file for it.